playbooks/tasks/create_interfaces.yml

```diff
@@ -75,6 +71,23 @@
     backup: no
   when: ansible_distribution == 'CentOS' or ansible_distribution == 'RedHat'
   with_items: "{{ interfaces }}"
-  notify: Reboot
+
+- name: Find all interface configuration scripts
+  find:
+    paths: /etc/sysconfig/network-scripts/
+    patterns: 'ifcfg-*'
+  register: ifcfg_scripts
+
+- name: Disable any interfaces not defined in inventory
```

Comment:
I'm ok with this for the moment, but I'm leary of disabling all interfaces not defined in inventory.  (what if another one was configured by something else or someone is rerunning post-deploy.

Reply:
I think we should be able to look at the MAC address. Let me take a close look tonight and see if I can add some logic.

Reply:
I think the change is going to be a little more complicated than expecting. I'm going to go ahead and submit this PR and open a new issue to track the change you suggested.